sites/partners/src/pages/index.tsx

```diff
@@ -304,6 +350,66 @@ export default function ListingsList() {
           />
         </article>
       </section>
+
+      <Dialog
+        isOpen={listingSelectModal}
+        ariaLabelledBy="listing-select-dialog-header"
+        ariaDescribedBy="listing-select-dialog-content"
+        onClose={() => setListingSelectModal(false)}
+      >
+        <Form id="listing-select-form" onSubmit={handleSubmit(onSubmit)}>
+          <Dialog.Header id="listing-select-dialog-header">
+            {"Add listing dialog header"}
+          </Dialog.Header>
+
+          <Dialog.Content id="listing-select-dialog-content">
+            {"Add listing dialog content"}
```

Comment:
suggestion: I know this is filler text until we fully implement this. So I'm putting a note here on what I think it should contain. It should explicitly reference that the jurisdiction cannot be changed after the fact.